### ISSUE TITLE
Can't delete model instances from admin list view that opens from the toolbar

### ISSUE THREAD

**Issue by creimers** [django-cms/django-cms#6495]
### Summary

I can't delete model instances from the list view that opens from the toolbar. In the "normal" admin list view it does work. From the toolbar opened detailview of the model instance it also works.

### Expected behaviour

I'm expecting to be able to delete model instances from the toolbar list view.

### Actual behaviour

When confirming the deletion, the modal goes blank. The following error appears in the Javascript console:

```
Uncaught TypeError: e.match is not a function
    at HTMLAnchorElement.<anonymous> (bundle.toolbar.min.js:1)
    at HTMLAnchorElement.dispatch (bundle.toolbar.min.js:1)
    at HTMLAnchorElement.g.handle (bundle.toolbar.min.js:1)
```

### Environment

* Python version: 3.6
* Django version: 1.11
* django CMS version: 3.5.2

This is happending both in the Firefox and Chrome (latest at time of writing).

**Comment by vxsx** [comment]
Could you provide an example project to reproduce?

**Comment by creimers** [comment]
@vxsx You got it: https://github.com/creimers/djangocms-delete-error

**Comment by vxsx** [comment]
So this happens due to the way browsers attach named inputs to forms. In your case form.action returns an input[name=action] instead of it's action property. I'll add a workaround

**Comment by creimers** [comment]
I can confirm that it also fixes the problem for me. Thanks!

**Comment by Aiky30** [comment]
Assuming fixed and closing this issue due to the feedback of fixed and seeing that it was released in 3.5.
